docs-source/content/userguide/introduction/introduction.md

```diff
@@ -57,10 +75,12 @@ WebLogic Server and the WebLogic Kubernetes operator are certified and supported
 
 ### OpenShift
 
-Operator 2.0.1+ is certified for use on OpenShift Container Platform 3.11.43+, with Kubernetes 1.11.5+.  OpenShift 4 certification is currently in progress.
+Operator 2.0.1+ is certified for use on OpenShift Container Platform 3.11.43+, with Kubernetes 1.11.5+.  
+
+Operator 2.5.0+ is certified for use on OpenShift Container Platform 4.3.0+ with Kubernetes 1.16.2+.
 
-When using the operator in OpenShift, the `anyuid` security context constraint is required to ensure that WebLogic containers run with a UNIX UID that has the correct permissions on the domain filesystem.
-For more information, see [OpenShift]({{<relref "/security/openshift.md">}}) in the Security section.
+When using the operator in OpenShift, a security context constraint is required to ensure that WebLogic containers run with a UNIX UID that has the correct permissions on the domain file system.
+This could be either the `anyuid` SCC or a custom one that you define for user/group 1000. For more information, see [OpenShift]({{<relref "/security/openshift.md">}}) in the Security section.
```

Comment:
This line is fine but I would suggest to use formatting like we have elsewhere in doc when referring to unix ids, my suggested change is:

"... or a custom one that you define for UID/GID of `1000`"

You see this syntax in first line at https://oracle.github.io/weblogic-kubernetes-operator/security/openshift/

Perhaps not in this change but was there a plan to update the OpenShift page with an example of a custom security context? Not sure it's needed just wondering,,,

Reply:
fixed

Reply:
yes i want to update that other page too, but have not done that in this pr